apis/core/v1alpha1/flagsourceconfiguration_types.go

```diff
@@ -45,7 +45,7 @@ const (
 	defaultSocketPath                string           = ""
 	defaultEvaluator                 string           = "json"
 	defaultImage                     string           = "ghcr.io/open-feature/flagd"
-	defaultTag                       string           = "v0.3.1"
+	defaultTag                       string           = "INPUT_FLAGD_VERSION"
```

Comment:
Could you please add a comment so that this isn't accidently changed in the future?

Reply:
I did this.